tests/Twig/MainExtensionTest.php

```diff
@@ -74,6 +74,6 @@ public function testGetAssetUrl(): void
             'http://lcl.doctrine-project.org',
         );
 
-        self::assertSame('http://lcl.doctrine-project.org/js/main.js?de1272', $url);
+        self::assertStringStartsWith('http://lcl.doctrine-project.org/js/main.js', $url);
```

Comment:
Maybe you can use a regex for the assertion in this test. We expect an assert hash here.